test/Microsoft.AspNetCore.Razor.Language.Test/DefaultRazorIRLoweringPhaseTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.Language.CodeGeneration;
+using Microsoft.AspNetCore.Razor.Language.Intermediate;
+using Microsoft.AspNetCore.Testing;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public class DefaultRazorIRLoweringPhaseTest
+    {
+        [Fact]
+        public void Execute_ThrowsForMissingDependency_SyntaxTree()
```

Comment:
Was shocked we didn't have a test for this to start so added this method for consistency. It's unrelated to this PR.